bucket/imhex.json

```diff
@@ -9,14 +9,18 @@
             "hash": "996ff7a1f26b40ced225a9d3cc7d9b695ea389895bc2bbba7734c39fc5044e2a"
         }
     },
-    "pre_install": "if (!(Test-Path \"$persist_dir\\imgui.ini\")) { New-Item \"$dir\\imgui.ini\" | Out-Null }",
     "shortcuts": [
         [
             "ImHex.exe",
             "ImHex"
         ]
     ],
-    "persist": "imgui.ini",
+    "persist": [
+        "config",
+        "recent",
+        "logs"
+    ],
+    "pre_uninstall": "if (Test-Path 'HKCU:\\SOFTWARE\\Classes\\*\\shell\\ImHex') { Remove-Item 'HKCU:\\SOFTWARE\\Classes\\*\\shell\\ImHex' -Recurse -Force }",
```

Comment:
What's this?

Reply:
The reason I have it there is so the context menu can be removed, but only if the user has activated from the **ImHex** application.
![(imhex)l0I6d1_12-17-2022](https://user-images.githubusercontent.com/101912712/208263226-9c1cfd79-fccd-4b40-a3f5-a3aba8e411c1.jpg)